specifyweb/frontend/js_src/lib/components/FormMeta/QueryTreeUsages.tsx

```diff
@@ -13,19 +13,16 @@ export function QueryTreeUsages({
 }): JSX.Element {
   return (
     <ProtectedAction action="execute" resource="/querybuilder/query">
-      <Link.Small
-        className={
-          resource.id === undefined
-            ? 'pointer-events-none !bg-white !ring-1 !ring-gray-400 dark:!bg-neutral-700 dark:!ring-neutral-500'
-            : ''
-        }
-        href={`/specify/query/fromtree/${resource.specifyModel.name.toLowerCase()}/${
-          resource.id
-        }/`}
-        target="_blank"
-      >
-        {formsText.findUsages()}
-      </Link.Small>
+      {resource.id === undefined ? null : (
```

Comment:
There is no point in rendering ProtectedAction if button is not displayed - ProtectedAction can display a dialog to the user if they don't have a permission, which is not helpful as there is no button, so no need for permission check.

Could you please modify the code by moving the ternary operator outside the ProtectedAction?

Reply:
optionally, if you want you could even move the input outside of this component